pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -566,6 +566,9 @@ self: super: {
   # https://github.com/alphaHeavy/lzma-enumerator/issues/3
   lzma-enumerator = dontCheck super.lzma-enumerator;
 
+  # Unpin some versions
+  lzma = appendPatch super.lzma ./patches/lzma-tests.patch;
```

Comment:
Could you please report this issue to @hvr at https://github.com/hvr/lzma/issues and add the link to that issue into the comment here? That makes it easier for other people to decide in the future whether the issue still exists.

Reply:
https://github.com/haskell-hvr/lzma/issues/8

Reply:
Please add that URL into the source code; it should replace the "Unpin some versions" comment.